ambari-metrics/ambari-metrics-assembly/pom.xml

```diff
@@ -266,10 +266,45 @@
                             <exclude>lib/*tests.jar</exclude>
                             <exclude>lib/findbugs*.jar</exclude>
                             <exclude>lib/jdk.tools*.jar</exclude>
+                            <exclude>lib/hadoop*.jar</exclude>
```

Comment:
I guess you are excluding here the jars from the hbase/lib that will be replaced later, right? You can add a comment why are those excluded.